### DISCUSSION TITLE
Unable to retrieve the Array of Items

### DISCUSSION THREAD

**Discussion by ganeshan102** [sensedeep/dynamodb-onetable#182] [Q&A] [answered]
let rules = await table.fetch(['Rule'], { pk: ruleId, SK: { begins_with: 'VS#R#' } })
rules.forEach(rule => {

    })

When i do iterate the above rules result, getting the below exception. I want to retrieve the list of items and which needs to be iterate.
This expression is not callable.
  Type 'AnyEntity[]' has no call signatures.ts(2349)

**Comment by mobsense** [comment, marked as answer]
Fetch does not return an array. It returns a map indexed by model type.

**Comment by ganeshan102** [comment]
Do we have any example in our repo?

**Reply by mobsense** [reply]
Not yet, you are welcome to write one and contribute.
